packages/lesswrong/components/common/TabNavigationMenu/EventsList.tsx

```diff
@@ -5,29 +5,26 @@ import { AnalyticsContext } from "../../../lib/analyticsEvents";
 const EventsList = ({currentUser, onClick}) => {
   const { TabNavigationEventsList } = Components
 
-  const lat = currentUser &&
-    currentUser.mongoLocation &&
-    currentUser.mongoLocation.coordinates[1]
-  const lng = currentUser &&
-    currentUser.mongoLocation &&
-    currentUser.mongoLocation.coordinates[0]
+  const lat = currentUser?.mongoLocation?.coordinates[1]
+  const lng = currentUser?.mongoLocation?.coordinates[0]
+
   let eventsListTerms: PostsViewTerms = {
     view: 'events',
     onlineEvent: false,
-    limit: 3,
+    limit: 1,
```

Comment:
This seems like it should be a DB setting, and also like we don't want to change it by accident